packs/feats/immortal-ruin.json

```diff
@@ -11,7 +11,7 @@
+            "value": "<p>Your commitment to ending all life earns you a measure of immortality, setting the stage for you to become a true Rider of the Apocalypse, possibly supplanting one of the Four. You gain the daemon, fiend, and unholy traits, and so does your apocalypse mount, but your fates are ever more entwined. As long as your apocalypse mount lives, you don't age and whenever your @UUID[Compendium.pf2e.conditionitems.Item.Doomed] or @UUID[Compendium.pf2e.conditionitems.Item.Dying] condition would increase to 4 or more, it doesn't increase. If you would be killed in any other way, you are instead reduced to 0 Hit Points, become @UUID[Compendium.pf2e.conditionitems.Item.Unconscious], and if your apocalypse mount is conscious, it instantly teleports to your side, picks you up, and attempts to flee the scene of danger.</p>\n<p>While you are conscious and within 30 feet of your apocalypse mount, it can't be killed. If your apocalypse mount is reduced to 0 Hit Points, it immediately stabilizes and its dying condition can't increase. If its doomed condition would increase to 4 or more, it doesn't increase. However, if you move more than 30 feet away from your apocalypse mount or go @UUID[Compendium.pf2e.conditionitems.Item.Unconscious] during this time, your apocalypse mount loses these benefits and might die.</p>\n<p>Finally, when a living creature within 60 feet of you dies, it must attempt a @Check[will|against:class-spell] saving throw against your class DC or spell DC, whichever is higher. This is an incapacitation effect. On a failure, that creature can't be brought back to life for 24 hours. On a critical failure, you can chose to have that creature's soul coalesce into a fist-sized soul gem (Hardness 2, HP 8) that appears in that creature's space. That creature's soul is trapped within the gem for 1 week, though you can make the gem permanent with a 10-minute ritual. Destroying the gem frees the soul within but doesn't return the deceased creature to life. If a caster attempts to return to life a creature whose soul is trapped within a soul gem, they fail unless they succeed at a @Check[religion|against:class-spell|rollerRole:target] check against your class DC or spell DC, whichever is higher. A success causes the soul gem to shatter, allowing the creature to be returned to life as normal for the spell. Daemons and other fiends prize these soul gems, and they're generally worth an amount relative to the level of a gem's captive soul.</p>"
```

Comment:
```suggestion
            "value": "<p>Your commitment to ending all life earns you a measure of immortality, setting the stage for you to become a true Rider of the Apocalypse, possibly supplanting one of the Four. You gain the daemon, fiend, and unholy traits, and so does your apocalypse mount, but your fates are ever more entwined. As long as your apocalypse mount lives, you don't age and whenever your @UUID[Compendium.pf2e.conditionitems.Item.Doomed] or @UUID[Compendium.pf2e.conditionitems.Item.Dying] condition would increase to 4 or more, it doesn't increase. If you would be killed in any other way, you are instead reduced to 0 Hit Points, become @UUID[Compendium.pf2e.conditionitems.Item.Unconscious], and if your apocalypse mount is conscious, it instantly teleports to your side, picks you up, and attempts to flee the scene of danger.</p>\n<p>While you are conscious and within 30 feet of your apocalypse mount, it can't be killed. If your apocalypse mount is reduced to 0 Hit Points, it immediately stabilizes and its dying condition can't increase. If its doomed condition would increase to 4 or more, it doesn't increase. However, if you move more than 30 feet away from your apocalypse mount or go @UUID[Compendium.pf2e.conditionitems.Item.Unconscious] during this time, your apocalypse mount loses these benefits and might die.</p>\n<p>Finally, when a living creature within 60 feet of you dies, it must attempt a @Check[will|against:class-spell|traits:incapacitation] saving throw against your class DC or spell DC, whichever is higher. This is an incapacitation effect. On a failure, that creature can't be brought back to life for 24 hours. On a critical failure, you can chose to have that creature's soul coalesce into a fist-sized soul gem (Hardness 2, HP 8) that appears in that creature's space. That creature's soul is trapped within the gem for 1 week, though you can make the gem permanent with a 10-minute ritual. Destroying the gem frees the soul within but doesn't return the deceased creature to life. If a caster attempts to return to life a creature whose soul is trapped within a soul gem, they fail unless they succeed at a @Check[religion|against:class-spell|rollerRole:target] check against your class DC or spell DC, whichever is higher. A success causes the soul gem to shatter, allowing the creature to be returned to life as normal for the spell. Daemons and other fiends prize these soul gems, and they're generally worth an amount relative to the level of a gem's captive soul.</p>"
```